docs/SETUP.md

```diff
@@ -0,0 +1,74 @@
+# Setup
+
+## Preparation
+
+In order to allow ecChronos to run there are a few tables that needs to be present.
+The keyspace name is configurable and is `ecchronos` by default.
```

Comment:
The keyspace created is 'ecchronos' but can be overwritten with configuration options (Link to options)